Update installation instructions on docs (#485)
SupertechLabs/client-superstaq: docs/source/get_started/installation.rst

Installation Guide
==================

> Superstaq can be easily installed for users from PyPI in your terminal. Please note that Python version 3.7 or higher is required.

Reviewer: Suggested rewording (up to you):
```suggestion
You can install Superstaq from PyPI in your terminal. Please note that Python version 3.7 or higher is required.
```

To install ``qiskit-superstaq``: 

.. code-block:: bash
    
    pip install qiskit-superstaq

To install ``cirq-superstaq``: 

.. code-block:: bash

    pip install cirq-superstaq

Installation for developers
---------------------------

> If you're interested in contributing to Superstaq, please check out our installiation instructions for developers `on Github <https://github.com/SupertechLabs/client-superstaq/blob/main/README.md#installation-for-development>`_!

Reviewer: Typo:
```suggestion
If you're interested in contributing to Superstaq, please check out our installation instructions for developers `on Github <https://github.com/SupertechLabs/client-superstaq/blob/main/README.md#installation-for-development>`_!
```
